modules/reference/pages/public-metrics-reference.adoc

```diff
@@ -497,6 +497,22 @@ Latency of produce/consume requests per broker. This duration measures from when
 
 ---
 
+=== redpanda_kafka_quotas_client_quota_throttle_time
+
+Client quota throttling delay, in seconds, per rule and quota type.
+
+*Type*: histogram
+
+---
+
+=== redpanda_kafka_quotas_client_quota_throughput
+
+Client quota throughput per rule and quota type.
```

Comment:
@pgellert This reference page also provides `*Labels*` info, can we provide it for the two new metrics?

Reply:
Nice! Yes, these are the labels on both:
```
quota_rule=("not_applicable" | "kafka_client_default" | "cluster_client_default" | "kafka_client_prefix" | "cluster_client_prefix" | "kafka_client_id")
quota_type=("produce_quota" | "fetch_quota" | "partition_mutation_quota")
```